docs/reference/enable-gpu.rst

```diff
@@ -1,46 +1,153 @@
+
+Ensure the Validator succeeded
+""""""""""""""""""""""""""""""
+
+Next we'll need to wait for the Validator Pod to succeed:
+
+.. code-block:: bash
+
+  echo "Waiting for the NVIDIA Operator validations to complete..."
+  while ! sudo microk8s.kubectl logs -n gpu-operator-resources -l app=nvidia-operator-validator -c nvidia-operator-validator | grep "all validations are successful"
+  do
+    sleep 5
+  done
+
+.. note::
+   It will take some seconds for the Pod to get initialized . The above command will
+   return a message like the following at the beginning:
+   ``Error from server (BadRequest): container "nvidia-operator-validator" in pod "nvidia-operator-validator-4rq5n" is waiting to start: PodInitializing``
+
+The expected output should be:
+
+.. code-block:: text
+
+   Error from server (BadRequest): container "nvidia-operator-validator" in pod "nvidia-operator-validator-4rq5n" is waiting to start: PodInitializing
+   ...
+   Error from server (BadRequest): container "nvidia-operator-validator" in pod "nvidia-operator-validator-4rq5n" is waiting to start: PodInitializing
+   all validations are successful
+
+Verify the DSS detects the GPU
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+  dss status
+
+Expected output:
+
+.. code-block:: bash
+
+  [INFO] MLflow deployment: Ready
+  [INFO] MLflow URL: http://10.152.183.74:5000
+  [INFO] GPU acceleration: Enabled (NVIDIA-GeForce-RTX-3070-Ti)
 
-* NVIDIA drivers installed on your machine.
-* The MicroK8s runtime configured to be able to use the Nvidia drivers.
+.. note::
 
-The `MicroK8s GPU add-on`_ is installing the NVIDIA Operator. The NVIDIA operator can either install or detect the NVIDIA drivers if they are already present on the host machine.
+  The GPU model `NVIDIA-GeForce-RTX-3070-Ti` will be different depending on your device.
 
-Before installing DSS:
+Launch GPU-enabled Notebook
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-1. Follow the MicroK8s documentation to `Install MicroK8s`_.
+At this point the DSS is fully configured to utilise the host's GPU. The next step will
+be to deploy a notebook that also contains CUDA runtimes, alongside with ML frameworks
+that can utilise the GPU.
 
-2. Enable MicroK8s GPU add-on.
+You can find a list of proposed images that include CUDA with the following command:
 
-   .. code-block:: bash
+.. code-block:: bash
 
-     sudo microk8s enable gpu
+   dss create --help | grep cuda
 
-3. Wait for GPU operator components to be ready. This can take around 5 minutes.
+You should see an output similar to this one:
 
-   .. code-block:: bash
+.. code-block:: bash
 
-     while ! sudo microk8s.kubectl logs -n gpu-operator-resources -l app=nvidia-operator-validator | grep "all validations are successful"
-     do
-       echo "waiting for validations"
-       sleep 5
-     done
+        - pytorch-cuda = kubeflownotebookswg/jupyter-pytorch-cuda-full:v1.8.0
```

Comment:
Question: are these hyphens correct? i.e., "- pytorch..."

Reply:
Yes, it's the actual output after running the above command and piping it to `grep`